test_View_Model_Unit.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+import View_Model as model
+from datetime import datetime
+
+"""
+Tests that items completed today go into recent_done_items list and the rest
+into the older_done_items list
+"""
+
+
+def test_recent_done_items_today():
+    today = datetime.today().strftime('%Y-%m-%d')
+    obj1 = model.ViewModel()
+    obj1._doneItems = [{'id': '123456', 'name': 'Sweep Yard', 'status': 'Done', 'date': today},
```

Comment:
We should avoid accessing private fields (marked with an `_`) directly - instead the ViewModel could either expose a setter for the done_items property, or maybe better the ViewModel could take a list of items in its constructor and use that to set the fields initially